lib/views/pages/account_settings/spending_limit.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:sun_flutter_capstone/views/widgets/input/input_field.dart';
+import 'package:sun_flutter_capstone/views/widgets/input/input_group.dart';
+import 'package:sun_flutter_capstone/views/widgets/template.dart';
+import 'package:sun_flutter_capstone/views/widgets/buttons/outline_button_text.dart';
+
+class SpendingLimit extends StatelessWidget {
+  const SpendingLimit({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final formKey = GlobalKey<FormState>();
+    final TextEditingController amountController = 
+    TextEditingController(text: '0.0');
```

Comment:
remove initial value and replace with hint text (upon clicking on text field, user has to erase the 0.0 value before being able to provide actual value)
![image](https://user-images.githubusercontent.com/86587091/165453216-ed5c3d3d-d2cc-437a-a1e2-f574704ebfca.png)